frontend/src/components/BigPopup/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Drawer } from '@mui/material';
+import Box from '@mui/material/Box';
+
+interface PopupProps {
+  isOpen: boolean;
+  togleClose: any; // this function reatern set of useState
+}
+export default function BigPopup(props: PopupProps) {
+  const { isOpen, togleClose } = props;
+
+  const fon: boolean = true;
```

Comment:
variable name "fon", use bg instead, and try not to use cyrillic naming convention